src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write your code here
+const populationArray = document.querySelectorAll('.population');
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+let totalPopulation = 0;
+
+populationArray.forEach((item) => {
+  const population = +item.textContent.replaceAll(',', '');
+
+  totalPopulation += population;
+});
+
+totalPopulationElement.textContent = totalPopulation.toLocaleString('en-US');
+
+averagePopulationElement.textContent = (
+  totalPopulation / populationArray.length
```

Comment:
Ensure that `populationArray.length` is not zero to avoid division by zero errors. Consider adding a check before performing the division.